components/src/NavBar/NavBar.js

```diff
@@ -120,8 +120,8 @@ class SmallNavBarNoState extends React.Component {
       menuData,
       menuState: { isOpen, handleMenuToggle, closeMenu },
       windowWidth,
-      smallBreakpoint,
-      smallScreen = windowWidth < smallBreakpoint,
+      breakpointCollapseLogo,
```

Comment:
this doesn't feel immediately obvious. i wonder if there's a different way to communicate this? "collapse logo" feels like an awkward phrase.